### PR TITLE
Feature/job view v2

### PR DESCRIPTION
**Pull request by utas-tajona**

### CODE REVIEW

File: ui/src/components/job/JobView.js [hunk 1]
```diff
@@ -4,14 +4,52 @@ import { useDispatch, useSelector } from 'react-redux';
...
+    'ABANDONED': function display() { return (<CheckCircleOutlineIcon />);},
+    'ERROR': function display() { return (<ErrorOutlineIcon />);}
+};
 
 const JobView = () => {
     const dispatch = useDispatch();
     const { id } = useParams();
     const job = useSelector(state => state.job.currentJob);
-    //const errors = useSelector(state => state.job.errors);
+    console.log(job);
```
Comment (nspool) [inline ui/src/components/job/JobView.js]: Remove console.log

File: ui/src/components/job/JobView.js [hunk 2]
```diff
@@ -20,43 +58,79 @@ const JobView = () => {
...
+                                            color="secondary"
+                                            label={'Extended Size'}
+                                            variant="outlined"
+                                        ></Chip>
+                                    </Grid>)}
+                            </Grid>
+                            <Divider style={{ marginTop: 15, marginBottom: 15 }}></Divider>
+                            <Grid>
+                                <Grid item>
+                                    <Typography>created by:  {job.creator.email}  at  {job.created} </Typography>
```
Comment (nspool) [inline ui/src/components/job/JobView.js:98]: Created should probably be capitalised.

File: ui/src/components/job/JobView.js [hunk 3]
```diff
@@ -63,7 +62,7 @@ const JobView = () => {
                             <Grid item lg={10} md={10} >
                                 <Typography variant="h5" color="primary">{job.reference}</Typography>
                             </Grid>
-                            <Grid container spacing={1} justify="flex-start">
+         a                   <Grid container spacing={1} justify="flex-start">
```
Comment (nspool) [inline ui/src/components/job/JobView.js:65]: Typo?

**Review by nspool — CHANGES_REQUESTED** [review]
Looks good except for a few minor things. Also the dates are displayed as ISO8601. Can they be formatted to display as 'YYYY-MM-DD HH:MM:ss', in local time?

File: ui/src/components/job/JobView.js (hunk 1 above)
Comment (utas-tajona) [reply ui/src/components/job/JobView.js]: console.log are breaking hte build it should be gone now

File: ui/src/components/job/JobView.js (hunk 3 above)
Comment (utas-tajona) [reply ui/src/components/job/JobView.js:65]: yes

**Comment by utas-tajona** [comment]
leaving the date as it is. IMO the format should be handle from the backend.

**Comment by nspool** [comment]
@utas-tajona when this is ready for review again please re-request review.

**Review by nspool — APPROVED** [review]

**Pull request merged**